server/go/pkg/server/database_test.go

```diff
@@ -32,3 +75,29 @@ func TestNewMysql(t *testing.T) {
 	assert.Nil(t, err)
 	assert.NotNil(t, dbconnection)
 }
+
+func TestSetRdbmsReplicaReadConsistencyValue(t *testing.T) {
+	dbconnection = nil
+
+	var err error
+	db, err := newMysql("root:secret@(localhost:%s)/mysql")
+	assert.Nil(t, err)
+	assert.NotNil(t, db)
```

Comment:
Two questions here:

`dbconnection` seems to be unused except above in `TestNewMysql`. TBH I can't even tell where this variable is coming from. But here in this test, you're using `db`. So why the reference to `dbconnection`?

Second, I don't think these first two assertions are necessary since they are just duplicating `TestNewMysql` above, verbatim.

Reply:
Ah ignore the first part on `dbconnection` -- I missed this at the package level initially. 😄

Reply:
I can remove the additional assertions, and only really had them in there to ensure the correct state for the test.